src/audio/module_adapter/module/generic.c

```diff
@@ -283,7 +283,7 @@ int module_reset(struct processing_module *mod)
 	/* module resets itself to the initial condition after prepare()
 	 * so let's change its state to reflect that.
```

Comment:
this comment should be corrected as well

Reply:
done, thanks!